### DISCUSSION TITLE
session in graphqlite

### DISCUSSION THREAD

**Discussion by dhiabenjaber** [thecodingmachine/graphqlite#479] [Q&A]
how to manage session and if i have a token how to passed with a query if there is an example . Thanks in advance

**Comment by oojacoboo** [comment]
GraphQLite doesn't deal with sessions or any session like tokens.  It's entirely up to you how to handle authentication, and even authorization layers.  Personally, I'd suggest having a look at the PASETO spec.  There is a nice lib for PHP available: https://github.com/paragonie/paseto.  API tokens are typically passed as a `Bearer` token in a request header.
